dev/integration_tests/release_smoke_test/pubspec.yaml

```diff
@@ -18,7 +18,11 @@ dev_dependencies:
   flutter_test:
     sdk: flutter
 
-  e2e: 0.6.0
+  e2e:
```

Comment:
This change is temporary for demo purposes only. After flutter/plugins#2873 is landed this will be updaged.